lib/src/nav-tabs/NavTabs.tsx

```diff
@@ -76,8 +76,8 @@ const DxcNavTabs = ({ iconPosition = "top", tabIndex = 0, children }: NavTabsPro
   return (
     <ThemeProvider theme={colorsTheme.navTabs}>
       <NavTabsContainer onKeyDown={handleOnKeyDown} role="tablist" aria-label="Navigation tabs">
-        <NavTabsContext.Provider value={contextValue}>{children}</NavTabsContext.Provider>
         <Underline />
+        <NavTabsContext.Provider value={contextValue}>{children}</NavTabsContext.Provider>
```

Comment:
There's no need for this change.

Reply:
This was done for the sake of homogeneity, as I realised that inside the normal `Tags`, the `Underline` was placed before the Context Provider, I know it makes no difference but I would like to follow the same structure

Reply:
Yes, I understood the change but, from my point of view, I don't know why we should show the underline before the tabs in the DOM tree. I know it doesn't affect since it has an absolute position but, for me, it makes more sense to have it after the tabs.